src/js/views/Reports/NamespaceKPIs.jsx

```diff
@@ -1,4 +1,4 @@
-import { Link } from 'react-router-dom'
+import { Link, useHistory, useLocation } from 'react-router-dom'
```

Comment:
Switch using `useNavigate` instead of `useHistory`.  See https://reactrouter.com/docs/en/v6/upgrading/v5#use-usenavigate-instead-of-usehistory

TL;DR -- `useNavigate()` returns a **function** that replaces `history.push(...)` so
```javascript
navigate = useNavigate()
navigate('/some/url')
```
instead of:
```javascript
history = useHistory()
history.push('/some/url')
```